src/univalue/test/object.cpp

```diff
@@ -13,11 +13,6 @@
 #include <string>
 #include <vector>
 
-#define BOOST_FIXTURE_TEST_SUITE(a, b)
-#define BOOST_AUTO_TEST_CASE(funcName) void funcName()
-#define BOOST_AUTO_TEST_SUITE_END()
-#define BOOST_CHECK(expr) assert(expr)
-#define BOOST_CHECK_EQUAL(v1, v2) assert((v1) == (v2))
 #define BOOST_CHECK_THROW(stmt, excMatch) { \
```

Comment:
This just seems to inline some checks while leaving others as-is. I can't really see a motivation to do this, especially as this is causing needless merge conflicts and inconsistent code.

I am fine removing the empty TEST macros, as they are unused. However, the other macros are used, so it would be better to keep them.

Reply:
> while leaving others as-is.

The others will be removed later. I'm planning on further consolidating the test code. I've just PR'd these changes first.